posthog/settings/web.py

```diff
@@ -13,8 +13,8 @@
 
 AXES_ENABLED = get_from_env("AXES_ENABLED", not TEST, type_cast=str_to_bool)
 AXES_HANDLER = "axes.handlers.cache.AxesCacheHandler"
-AXES_FAILURE_LIMIT = int(os.getenv("AXES_FAILURE_LIMIT", 5))
-AXES_COOLOFF_TIME = timedelta(minutes=15)
+AXES_FAILURE_LIMIT = int(os.getenv("AXES_FAILURE_LIMIT", 200))
```

Comment:
This feels just about high enough to start brute-forcing via some saved password lists. How about ~50?

Reply:
Set to 50.